crates/agglayer-storage/src/columns/mod.rs

```diff
@@ -6,6 +6,14 @@ pub enum CodecError {
     #[error(r#"Serialization error: {0}
         This is a critical bug that needs to be reported on `https://github.com/agglayer/agglayer/issues`"#)]
     Serialization(#[from] bincode::Error),
+
+    #[error(r#"Certificate encoded to an empty byte sequence.
+        This is a critical bug that needs to be reported on `https://github.com/agglayer/agglayer/issues`"#)]
+    CertEmpty,
+
+    #[error(r#"Unrecognized certificate storage format version {version}.
+        This is a critical bug that needs to be reported on `https://github.com/agglayer/agglayer/issues`"#)]
+    BadCertVersion { version: u8 },
```

Comment:
We can have this generic and reusable, I think.

```suggestion
    #[error(r#"Unrecognize storage format version {version}.
        This is a critical bug that needs to be reported on `https://github.com/agglayer/agglayer/issues`"#)]
    BadVersion { version: u8 },
```

Reply:
Okey, then we can use `BadCertificateVersion`